sdk/keyvault/Azure.Security.KeyVault.Certificates/src/DeleteCertificateOperation.cs

```diff
@@ -24,18 +24,15 @@ internal DeleteCertificateOperation(KeyVaultPipeline pipeline, Response<DeletedC
         {
             _pipeline = pipeline;
             _value = response.Value ?? throw new InvalidOperationException("The response does not contain a value.");
+            // The recoveryId is only returned if soft delete is enabled.
```

Comment:
Nit: can you add a blank line between 26 and 27. IIRC, this may cause some code style errors.